src/runtime_strings.js

```diff
@@ -63,12 +63,18 @@ var UTF8Decoder = typeof TextDecoder !== 'undefined' ? new TextDecoder('utf8') :
 #endif // TEXTDECODER
 #endif // TEXTDECODER == 2
 
-function UTF8ArrayToString(u8Array, idx) {
+/**
+ * @param {number} idx
+ * @param {number=} endIdx
```

Comment:
what does the `=` mean here? (perhaps something about it being optional?)

Reply:
Yeah, that annotates for Closure that the given parameter does not need to be passed in. Otherwise Closure would issue a warning that the function was called with different number of args that it was defined with.